connect-tests/Cargo.toml

```diff
@@ -7,7 +7,6 @@ description = "Rustls connectivity based integration tests."
 publish = false
 
 [features]
```

Comment:
nit: maybe drop this empty `[features]` section?

Reply:
Deleted.